code/datums/quirks/negative_quirks.dm

```diff
@@ -238,6 +238,42 @@
 	quirk_holder.clear_mood_event("family_heirloom_missing")
 	quirk_holder.clear_mood_event("family_heirloom")
 
+/datum/quirk/glassjaw
+	name = "Glass Jaw"
+	desc = "You have a very fragile jaw. Any sufficiently hard blow to your head might knock you out."
+	icon = "glass"
```

Comment:
Isn't this just a drinking glass? Gonna look weird, What about `bone-break`, do we use that?